test/stdexec/algos/other/test_execute.cpp

```diff
@@ -49,16 +50,17 @@ TEST_CASE("execute works with schedulers that need to be triggered manually", "[
 
 TEST_CASE("execute works on a thread pool", "[other][execute]") {
   exec::static_thread_pool pool{2};
-  bool called{false};
+  std::atomic<bool> called{false};
   {
     // launch some work on the thread pool
-    ex::execute(pool.get_scheduler(), [&] { called = true; });
+    ex::execute(pool.get_scheduler(), [&] { called.store(true, std::memory_order_relaxed); });
```

Comment:
Why store and load with relaxed semantics here, but sequential consistency elsewhere?

Reply:
I've started with care, then noticed that other places might requrie ordering and switched to sequential consistency not to spend much time :)